packages/service/src/domain/commands/processors/tag-search-command.ts

```diff
@@ -0,0 +1,35 @@
+import XSoundsService from '../../x-sounds/x-sounds-service';
+import Command from '../model/command';
+import CommandDefinition from '../model/command-definition';
+import CommandProcessingResponse from '../model/command-processing-response';
+
+async function tagSearchCommandProcessor(command: Command): Promise<CommandProcessingResponse> {
+  const tagName = command.rawArgs.trim();
+  if (!tagName) {
+    throw new Error('Podaj nazwę tagu');
+  }
+
+  const sounds = await XSoundsService.instance.getAllByTag(tagName);
+  const tagListText = sounds.map((sound) => sound.name).join('\n');
+
+  return {
+    messages: [
+      { type: 'HEADER', text: `Dźwięki z tagiem ${tagName}` },
+      { type: 'MARKDOWN', text: tagListText },
+    ],
+    isVisibleToIssuerOnly: false,
+  };
+}
+
+const tagSearchCommandDefinition: CommandDefinition = {
+  key: 'tag-search',
+  shortKey: 'ts',
+  processor: tagSearchCommandProcessor,
+  helpMessage: 'Wyszukuje dźwięki z danym tagiem',
+  helpUsages: [
+    '<tag name>',
+    'fun stuff',
```

Comment:
I thought that tags with space are disallowed

Reply:
I went above and beyond to make spaces possible.


(I've copy-pasted code from `addx` command)